generate_docs.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+# html format
+# --force deletes previous docs
+# -o is the output dir
+os.system("pdoc otri --html --force -o doc")
```

Comment:
Codacy found an issue: [Starting a process with a shell: Seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5804020)